openshift/README.md

```diff
@@ -30,3 +31,14 @@ CentosCI will build your image and comment on the PR with an image name and tag.
 Your image is available in the registry: docker pull registry.devshift.net/fabric8-analytics/worker-scaler:SNAPSHOT-PR-25
 
 To update your dev deployment use [oc edit](https://docs.openshift.com/container-platform/3.4/cli_reference/basic_cli_operations.html#edit) from the command line or editor in web interface.
+
+## Run E2E test agaist your deployment
```

Comment:
I am missing information on how to actually run them :) I don't expect people to read the whole README file in integration-tests/ directory. This should be straightforward, i.e. do this-this-this and bum! the tests are running.

Reply:
Indeed I have marked this as WIP. Updating it continuously...